pkg/k8sclient/podwatcher.go

```diff
@@ -258,7 +258,7 @@ func (this *PodWatcher) podWorker() {
 				switch pod.State {
 				case PodPending:
 					glog.V(2).Info("PodPending ", pod.Identifier)
-					PodsCond.L.Lock()
+					PodsMux.Lock()
```

Comment:
the name `PodMux` is ambiguous with `Pod Multiplexer`, could we simply call `podMutex` to be clearer?

Reply:
You're right, I will modify it.